samples/snapshot/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>pubsublite-spark-snapshot</artifactId>
   <packaging>jar</packaging>
-  <name>Google Pub/Sub Lite Spark Connector Snapshot Samples</name>
+  <name>Google Pub/Sub Lite Spark Connector Snapshot</name>
```

Comment:
What's the reason to remove "Samples" here? I'd leave it in.

Reply:
I am keeping snippets the same... ok, let me keep it.